components/layout/fragment.rs

```diff
@@ -996,6 +1002,7 @@ impl Fragment {
             specific: info,
             inline_context: self.inline_context.clone(),
             pseudo: self.pseudo.clone(),
+            flags: FragmentFlags::empty(),
```

Comment:
Have you verified that adding these flags does not change the size of `Fragment`?